src/main/webapp/onelive/index.html

```diff
@@ -277,13 +285,75 @@ <h5 class="title">
+                    <span class="small text-muted mr-4 text-uppercase">{{month}}, {{day}}</span>
+                    <span class="mr-5"> {{startTime}} - {{endTime}}</span>
+                    <span class="font-weight-bold ml-1">{{summary}}</span>
+                </div>
+            </div>
+        </a>
+    </div>
+    {{/data}}
+</script>
+
 <script>
     $(function () {
         loadNavAndFooter('/assets/content/static');  //relative path to content directory
+        loadUpcomingEvents(); //load upcoming events
         loadProfiles(); // load profiles
         loadFaqComponent();
     });
 
+    //function to load upcoming events
+    function loadUpcomingEvents() {
+        $.ajax({
+            type: 'GET',
+            url: 'https://sef-dataholder.herokuapp.com/calendar/onelive',
+            dataType: 'json',
+            success: function (payload) {
+                if (payload.items == null) {
+                    $("#section-events").hide();
+                } else {
+                    for (let i = 0; i < payload.items.length; i++) {
+                        //to get img link (optional)
+                        /*let fileId = payload.items[i].attachments[0].fileId;
+                        payload.items[i].imgLink = "https://drive.google.com/uc?export=view&id=" + fileId;*/
+
+                        //get fb link and remove it from description (split the part of the description where the fb link is added)
+                        let description = payload.items[i].description;
+                        payload.items[i].facebookLink = description.substring(description.lastIndexOf('<a href') + 9, description.lastIndexOf('" id='));
+                        payload.items[i].description = description.substring(0, description.lastIndexOf('<a href'));
+                        //get date & time of the event
+                        let start = new Date(payload.items[i].start.dateTime);
+                        let end = new Date(payload.items[i].end.dateTime);
+                        payload.items[i].day = start.toLocaleDateString([], {weekday: 'short'});
+                        payload.items[i].month = start.toLocaleDateString([], {month: 'short'});
+                        payload.items[i].date = start.toLocaleDateString([], {day: '2-digit'});
+                        payload.items[i].startTime = start.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit', hour12: false});
+                        payload.items[i].endTime = end.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit', hour12: false});
+                        //Mustache render
+                        let content = Mustache.render($("#upcoming-events-template").html(), {"data": payload.items});
+                        $("#upcoming-events").html(content);
+                    }
+                }
+            },
+            error: function () {
+                console.log("Error occurred while loading upcoming events");
```

Comment:
Console logs should not be in the code.

Reply:
The event in the calendar is not in the suitable format. It was added as full day event. Have to add the events as limited time events

> The preview isn't working
> ![Screenshot 2020-04-11 at 15 18 03](https://user-images.githubusercontent.com/33048395/79040659-a7336500-7c07-11ea-897f-05398f754d9a.jpg)

The event in the calendar is not in the suitable format. It was added as full day event. Have to add the events as limited time events

Reply:
Ok